DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
ERROR:root:Could not understand audio.
ERROR:django.request:Internal Server Error: /verify_voice/
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
DEBUG:root:Speech recognition successful.
WARNING:django.request:Bad Request: /verify_voice/
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
ERROR:root:Could not understand audio.
ERROR:django.request:Internal Server Error: /verify_voice/
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
DEBUG:root:Speech recognition successful.
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
DEBUG:root:Speech recognition successful.
WARNING:django.request:Bad Request: /verify_voice/
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
DEBUG:root:Speech recognition successful.
WARNING:django.request:Bad Request: /verify_voice/
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
DEBUG:root:Speech recognition successful.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
DEBUG:root:Received POST request to /verify_voice/
DEBUG:pydub.converter:subprocess.call(['ffmpeg', '-y', '-read_ahead_limit', '-1', '-i', 'cache:pipe:0', '-acodec', 'pcm_s32le', '-vn', '-f', 'wav', '-'])
DEBUG:root:Speech recognition successful.
WARNING:django.request:Not Found: /verify_voice_with_assemblyai/
DEBUG:httpx:load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG:httpx:load_verify_locations cafile='/home/d3vil/Wibit/env/lib/python3.11/site-packages/certifi/cacert.pem'
WARNING:django.request:Bad Request: /verify_voice/
WARNING:django.request:Bad Request: /verify_voice/
WARNING:django.request:Bad Request: /verify_voice/
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
DEBUG:httpx:load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG:httpx:load_verify_locations cafile='/home/d3vil/Wibit/env/lib/python3.11/site-packages/certifi/cacert.pem'
WARNING:django.request:Bad Request: /verify_voice_with_assemblyai/
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
DEBUG:httpx:load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG:httpx:load_verify_locations cafile='/home/d3vil/Wibit/env/lib/python3.11/site-packages/certifi/cacert.pem'
WARNING:django.request:Bad Request: /verify_voice_with_assemblyai/
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/project/urls.py changed, reloading.
DEBUG:httpx:load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG:httpx:load_verify_locations cafile='/home/d3vil/Wibit/env/lib/python3.11/site-packages/certifi/cacert.pem'
WARNING:django.request:Bad Request: /voice_with_assemblyai/
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
DEBUG:httpx:load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG:httpx:load_verify_locations cafile='/home/d3vil/Wibit/env/lib/python3.11/site-packages/certifi/cacert.pem'
WARNING:django.request:Bad Request: /voice_with_assemblyai/
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
INFO:django.utils.autoreload:/home/d3vil/Wibit/project/users/views.py changed, reloading.
